prov/efa/src/rxr/rxr_pkt_type_misc.c

```diff
@@ -535,6 +536,11 @@ void rxr_pkt_handle_receipt_sent(struct rxr_ep *ep,
 {
 }
 
+void rxr_pkt_handle_handshake_sent(struct rxr_ep *ep,
+				 struct rxr_pkt_entry *pkt_entry)
+{
```

Comment:
Set the flag `RXR_PEER_HANDSHAKE_SENT` here is preferred

Reply:
@wzamazon I posted the reason why we can not set flag here above.

Reply:
Copied here for your reference.
```
Also noted that setting RXR_PEER_HANDSHAKE_SENT flag in rxr_pkt_handle_handshake_sent can also cause mutiple handshake being sent, as rxr_pkt_handle_ctrl_sent will not be invoked if rxr_pkt_entry_send returns FI_EAGAIN.
https://github.com/ofiwg/libfabric/blob/master/prov/efa/src/rxr/rxr_pkt_cmd.c#L339-L348
So I leave it setting in rxr_pkt_post_handshake.
```

Reply:
I see your point. Even if handshake has not been sent, as long as it will be retried, we do not want to send a new handshake. In this case, the flag name `RXR_PKT_HANDSHAKE_SENT` is not accurate. Can you rename it something else like "SENT_OR_QUEUED". Also leave a comment.